packages/kbn-securitysolution-es-utils/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "declaration": true,
     "declarationMap": true,
-    "outDir": "target",
+    "emitDeclarationOnly": true,
+    "outDir": "target_types",
```

Comment:
After the include below did you want this like the others?

```json
  "exclude": [
    "**/__fixtures__/**/*"
  ]
```

Reply:
we don't have fixtures on that package for the moment being I think 😄